build/tasks/browserify.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var browserify = require('browserify');
+var derequire = require('derequire'); // This is a silly thing to type
+
+var bannerTemplate = '/*! p5.js v<%= pkg.version %> <%= grunt.template.today("mmmm dd, yyyy") %> */';
+
+module.exports = function(grunt) {
+
+  var libFilePath = path.join(process.cwd(), 'lib/p5.js');
```

Comment:
Used `require.resolve` for the `srcFilePath` app entry point, and `path.resolve` for the `libFilePath` build destination file (which will not yet exist until built in a clean environment, causing `require.resolve` to fail)